### DISCUSSION TITLE
Switch Lite buttons as second controller

### DISCUSSION THREAD

**Discussion by Vetal580** [ndeadly/MissionControl#871] [Q&A]
I connected DS4 to my Switch Lite. Everything works fine. But in games for 2+ players only this DS4 gamepad available to choose. Is there a way to use external controller and Lite controls as second gamepad?

**Comment by ndeadly** [comment]
Good question. I'd never considered this case before. It might be possible to patch `hid` or similar to allow it. Would require some investigation.

**Reply by Vetal580** [reply]
Got it. I would be grateful if you could look into it when possible.
